scapy/layers/ppp.py

```diff
@@ -581,7 +581,7 @@ def dispatch_hook(cls, _pkt=None, *_, **kargs):
             code = orb(_pkt[0])
         elif "code" in kargs:
             code = kargs["code"]
-            if isinstance(code, basestring):
+            if isinstance(code, str):
```

Comment:
I think `six.string_types` would be more appropriate here.

Reply:
forgot to push...

Reply:
(I was talking about this comment @gpotter2)

Reply:
(and I mean it, it's not "outdated")